### PR TITLE
little tweaks to the .css

### PR DESCRIPTION
**Pull request by vishus12**

### CODE REVIEW

File: newZealand.html [hunk 1]
```diff
@@ -25,7 +24,7 @@ <h1>New Zealand</h1>
             <div class="agency">
                 <h2>Get Lost Travel Agency</h2>
             </div>
-            <img class="logo vish" src="./images/travel_logo.png" alt="Travel Logo">
+            <img class="logo vish" src="./images/newzealand_logo.png" alt="newzealand_logo">
         </header>
     </div>
     <main class="kiwiContainer">
```
Comment (jacksonrgoodman) [inline newZealand.html:30]: needs` <ul>` on next line, to complete the "unordered list" thought, before continuing.

File: newZealand.html [hunk 2]
```diff
@@ -41,7 +40,7 @@ <h2>People</h2>
                 <li>NEIL FINN – TE AWAMUTU</li>
                 <li>STEVEN ADAMS – ROTORUA</li>
                 <li>FLIGHT OF THE CONCHORDS – WELLINGTON</li>
-                </ul>
+            </ul>
 
 
         </div>
```
Comment (jacksonrgoodman) [inline newZealand.html:46]: needs `<ul>` on next line, to complete the "unordered list" thought, before continuing.

File: newZealand.html [hunk 3]
```diff
@@ -57,7 +56,7 @@ <h2>Cities</h2>
                 <li>Lower Hutt</li>
                 <li>Dunedin</li>
                 <li>Palmerston North</li>
-                </ul>
+            </ul>
         </div>
         <div class="kiwi">
             <h2>Landmarks</h2>
```
Comment (jacksonrgoodman) [inline newZealand.html:62]: needs ` <ul>` on next line, to complete the "unordered list" thought, before continuing.

**Review by jacksonrgoodman — APPROVED** [review]
left notes

**Review by Chadwell-Clark — APPROVED** [review]

**Pull request merged**